src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/ProgressBar/ProgressBarHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Xamarin.Platform.Handlers
+{
+	public partial class ProgressBarHandler
+	{
+		public static PropertyMapper<IProgress, ProgressBarHandler> ProgressMapper = new PropertyMapper<IProgress, ProgressBarHandler>(ViewHandler.ViewMapper)
+		{
+			[nameof(IProgress.Progress)] = MapProgress,
+			[nameof(IProgress.ProgressColor)] = MapProgressColor,
+#if __IOS__
+			[nameof(IView.BackgroundColor)] = MapBackgroundColor
+#endif
```

Comment:
Why is this iOS only?

Reply:
Even if this is just for iOS, how will this translate to me when I am customizing the handler?

Reply:
I see Android has a progressBackgroundTint which may be this, but for android? We may want to override this.

Reply:
Because in the ProgressBar Renderer we set the `TrackTintColor` property in iOS. BUT...this a good point. Now, in Xamarin.Forms the BackgroundColor property behavior is different between iOS and Android. Must we match the behavior (use `progressBackgroundTint`)?

Reply:
Now, Android and iOS have the same behavior.